examples/3d/normalsExample/src/ofApp.cpp

```diff
@@ -130,7 +130,7 @@ void ofApp::draw(){
 
     ofSetColor(255);
     ofDrawBitmapString("press any key or mouse button to disable mesh normals", 20,20);
-    ofDrawBitmapString("light", cam.worldToScreen(light.getGlobalPosition()) + ofPoint(10,0));
+    ofDrawBitmapString("light", cam.worldToScreen(light.getGlobalPosition()) + glm::vec3(20,0,0));
```

Comment:
Any reason why 10 changed to 20?

Reply:
not sure, did this a long ago, probably it looks better???

Reply:
just checked. otherwise it overlaps with the sphere and can not be read properly